Automattic/simplenote-ios
# Feature/finalize release

This PR adds a script to automate some steps of the release finalization process.
The PR builds on top of https://github.com/Automattic/simplenote-ios/pull/356 and should be merged after that one.

# To Test:
1. Checkout a new `release/x.x` branch. Ex: `release/4.999`.
2. Push the branch to GItHub with the `-u` option.
3. Run `bundle exec fastlane finalize_release`.
4. When prompted, verify that the system recognises the right version and confirm.
5. Verify that the script completes without errors.
6. Verify that the translations are downloaded.
7. Verify that the `4.8` tag is created.
8. Verify that all the changes are pushed.

# Clean up
9. Delete the `4.8` tag from your local repository and from GitHub.
10. Delete the test branch from your local repository and from GitHub.
